README.md

```diff
@@ -38,5 +37,16 @@ This application can be configured using two environment variables:
     This could be for example `http://kratos.my-app.com/`.
 - `BASE_URL`: The base url of this app. If served e.g. behind a proxy or via GitHub pages this would be the path, e.g.
     `/kratos-selfservice-ui-node/`
+
+### With Oathkeeper (JWT)
+
+- `JWKS_URL`: The URL to be used to check if the signature of the incoming ID token is valid.
+
+### Without Oathkeeper (Cookie)
+
+- `KRATOS_WITH_OATHKEEPER`: This flag indicates that this app is used without Oathkeeper. In that case, the server proxies directly Kratos public API. `KRATOS_BROWSER_URL` have to be set to this application base URL.
```

Comment:
Maybe we rename this to `SECURITY_MODE=standalone|jwt`? That way we can add more modes later on (e.g. using HTTP Headers which would also expect something like oathkeeper). This would also make it clear that you could use another gateway (e.g. Kong) with some auth filter I think.

Also, now that I understand that `KRATOS_BROWSER_URL` is no longer required when using this mode, I think we can say that it's not required to set the env var here.

Maybe we could add one more line here which says that the Kratos API is proxied under `/self-service/...`. What do you think?